Islandora-CLAW/Alpaca
# Removing islandora-connector-broadcast

**GitHub Issue**: Resolves https://github.com/Islandora-CLAW/CLAW/issues/974

* Part of https://github.com/Islandora-CLAW/CLAW/issues/950, really

# What does this Pull Request do?

Removes `islandora-connector-broadcast` since we've switched to pumping directly to queues from Drupal.

# How should this be tested?

`./gradlew build` should run successfully and never mention `islandora-connector-broadcast`.  The features file that gets written out to `karaf/build/resources/features.xml` should not contain a `islandora-connector-broadcast` entry.

# Interested parties
@Islandora-CLAW/committers
